presto-main/src/test/java/com/facebook/presto/sql/planner/iterative/rule/TestPickTableLayout.java

```diff
@@ -60,7 +63,7 @@
     @BeforeClass
     public final void setUp()
     {
-        tester = new RuleTester();
+        tester = new RuleTester(true);
```

Comment:
maybe instead of this you could make `RuleTester` to accept `QueryRunner` in constructor, then `BaseRuleTest` could have a protected method to create query runner, which could be overwritten by this test. 
What do you think?

Reply:
tests that need to do other kinds of set up become more complicated because they don't extend BaseRuleTest, so then you need to create the sessions/QueryRunners for those tests separately there.  This applies to `TestTransformExistsApplyToScalaraLateralJoin` and `TestTransforCorrelatedScalarAggregationToJoin`.